frontend/www/js/omegaup/arena/submissions.ts

```diff
@@ -16,6 +20,18 @@ interface RunSubmit {
   runs: types.Run[];
 }
 
+interface SubmissionRequestWithExtraData {
+  redirect: string;
+  request: { guid: string; isAdmin: boolean; problemAlias: string };
+  target: problem_Details;
```

Comment:
```suggestion
  request: SubmissionRequest;
```

así se mantiene la petición original "intacta".

Reply:
Listo